### PR TITLE
Spacesuit & Workboot style balance

### PR DESCRIPTION
**Pull request by Kurgis**
## About The Pull Request
Some misc. style changes I've been gathering on my repo.

## Why It's Good For The Game
Balance changes are never good.

## Changelog
:cl:
balance: Workboots, captain's armor, and red/tan space suits are now more stylish than before.
balance: Captain's armor is somewhat harder to breach, and contractor space suits have become less stiff.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -634,6 +634,7 @@
 	)
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	style_coverage = COVERS_TORSO|COVERS_UPPER_ARMS|COVERS_UPPER_LEGS
+	accompanying_object = /obj/item/clothing/head/armor/faceshield/paramedic
```
Comment (vode-code) [inline code/modules/clothing/suits/armor.dm:637]: if you're going to do that, you should probably make sure this doesn't make there be two helmets everywhere this is.

File: code/modules/clothing/spacesuits/syndi.dm [hunk 2]
```diff
@@ -45,11 +47,11 @@
 	icon_state = "syndicate_tan"
 	item_state = "syndi_helm_tan"
 	desc = "A knockoff tan helmet sporting clean lines and durable plating."
+	style = STYLE_NONE
```
Comment (vode-code) [inline code/modules/clothing/spacesuits/syndi.dm:50]: why would a basic knockoff tan helm have comparable style to the captain's "most fashionable" helm?

File: code/modules/clothing/spacesuits/syndi.dm [hunk 3]
```diff
@@ -15,6 +15,7 @@
 	)
 	siemens_coefficient = 0.4
 	spawn_blacklisted = TRUE
+	style = STYLE_LOW
```
Comment (vode-code) [inline code/modules/clothing/spacesuits/syndi.dm:18]: No.

File: code/modules/clothing/spacesuits/captain.dm [hunk 4]
```diff
@@ -13,6 +13,7 @@
 		rad = 50
 	)
 	siemens_coefficient = 0.7
+	style = STYLE_NONE
```
Comment (vode-code) [inline code/modules/clothing/spacesuits/captain.dm:16]: if you want to make voidsuits have style, you must make sure account that the Captain always has the most style here, as is everywhere else.

**Review by vode-code — CHANGES_REQUESTED** [review]
I don't care about any of the changes I saw and didn't comment on, but the voidsuit choices are an atrocity.

File: code/modules/clothing/spacesuits/syndi.dm [hunk 5]
```diff
@@ -36,7 +37,8 @@
 	spawn_blacklisted = TRUE
 	accompanying_object = /obj/item/clothing/head/space/syndicate
 	slowdown = LIGHT_SLOWDOWN
-	stiffness = HEAVY_STIFFNESS
+	stiffness = LIGHT_STIFFNESS
```
Comment (Humonitarian) [inline code/modules/clothing/spacesuits/syndi.dm:40]: Is this an intended change?

Comment (DiaFRAME444) [reply code/modules/clothing/spacesuits/syndi.dm:40]: From the changelog
balance: Captain's armor is somewhat harder to breach, and contractor space suits have become less stiff.

**Comment by Humonitarian** [comment]
#7990 will affect this

**Comment by Kurgis** [comment]
This has gone stale.

**Pull request closed without merging**